ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,417 @@
+                </div>
+              </FlexColumn>
+            </FlexRow>
+            <FlexRow style={{marginTop: '20px'}}>
+              <CheckBox style={{margin: '15px', height: '1.063rem', width: '1.063rem'}}
+                        checked={nihFunded === true}
+                        onChange={() => setNihFunded(false)}/>
+              <FlexColumn style={{marginTop: '9px', marginLeft: '15px', marginBottom: '15px'}}>
+                <div style={styles.textHeader}>NIH-funded Research.</div>
+                <div style={styles.textNormal}>My research is funded by the National Institute of Health, (NIH).
+                  NIH funded research is eligible for discounted cloud rates through
+                  the NIH STRIDES initiative.
+                </div>
+              </FlexColumn>
+            </FlexRow>
+          </FlexColumn>
+          <FlexRow style={{marginTop: '100px', justifyContent: 'space-between'}}>
+            <Button type='secondary'
+                    style={{fontWeight: 400, padding: '0 18px', height: '40px'}}
+                    onClick={() => {setCurrentStep(1); }}>
+              Back
+            </Button>
+            <FlexRow style={{justifyContent: 'flex-end'}}>
+              <Button type='secondary'
+                      style={{fontWeight: 400, padding: '0 18px', height: '40px', marginRight: '10px'}}
+                      onClick={() => onClose()}>
+                Cancel
+              </Button>
+              <Button data-test-id='next-button'
+                      type='primary'
+                      style={{fontWeight: 400, padding: '0 18px', height: '40px', width: '93px'}}
+                      disabled={useCreditCard === null}
+                      onClick={() => {setCurrentStep(3); }}>
+                Next
+              </Button>
+            </FlexRow>
+          </FlexRow>
+        </FlexColumn>
+      </ModalFooter>)
+      ], [
+        3, () => (<ModalFooter data-test-id='step-3-modal' style={{marginTop: 0, justifyContent: 'flex-start'}}>
+        <FlexColumn style={{width: '100%'}}>
+          <div style={styles.textHeader}>Please review your information</div>
+          <TextColumn>
+            <FlexRow  style={{marginTop: '15px'}}>
+              <div style={{width: '170px'}}>Name: </div>
+              <div data-test-id='user-full-name-text'>{givenName + ' ' + familyName}</div>
+            </FlexRow>
+            <FlexRow  style={{marginTop: '5px'}}>
+              <div style={{width: '170px'}}>Phone mumber: </div>
+              <div data-test-id='user-phone-number-text'>{phoneNumber}</div>
+            </FlexRow>
+            <FlexRow  style={{marginTop: '5px'}}>
+              <div style={{width: '170px'}}>Contact email: </div>
+              <div data-test-id='user-contact-email-text'>{contactEmail}</div>
+            </FlexRow>
+            <FlexRow  style={{marginTop: '5px'}}>
+              <div style={{width: '170px'}}>Researchallofus ID: </div>
```

Comment:
This seems like a mock issue, presumably this should be `researchallofus.org ID`